alt_e2eshark/onnx_tests/helper_classes.py

```diff
@@ -23,6 +25,50 @@
 """This file contains several helpful child classes of OnnxModelInfo."""
 
 
+class HfDownloadableModel(OnnxModelInfo):
+    """This class should be used to download models from Huggingface model hub."""
+
+    def __init__(self, full_model_path, task_name, name, onnx_model_path):
+        opset_version = 21
+        self.model_repo_path = full_model_path.replace("hf_", "")
+        self.task = task_name
+        super().__init__(name, onnx_model_path, opset_version)
+
+    def export_model(self):
+        model_dir = str(Path(self.model).parent)
+        # TODO: Use Python API instead of CLI?
+        opt_cli_command = f"optimum-cli export onnx --model {self.model_repo_path} --task {self.task} --monolith --framework pt {model_dir}"
+        cli_output = subprocess.run(opt_cli_command.split(' '), capture_output=True)
+        # TODO: Find a cleaner way to exit than an assert.
+        assert cli_output.returncode == 0, f"Failed to run optimum-cli:\n{cli_output.stderr.decode()}"
```

Comment:
I don't want to crash the test runner if this fails. Let's raise an exception instead. E.g. :
```suggestion
        if cli_output.returncode == 0:
           raise RuntimeError(f"Failed to run optimum-cli:\n{cli_output.stderr.decode()}")
```